### PR TITLE
Optimization and fixes

### PR DESCRIPTION
**Pull request by blackpanther1881**

### CODE REVIEW

File: src/utils/transactions.js [hunk 1]
```diff
@@ -155,20 +155,23 @@ async function MakeIBCTransferMsg(channel, fromAddress, toAddress, amount, timeo
...
-        revisionNumber: clientStateResponseDecoded.latestHeight.revisionNumber
-    };
-
-    const consensusStateResponse = await ibcExtension.ibc.channel.consensusState(port,channel,
-        clientStateResponseDecoded.latestHeight.revisionNumber.toInt() , clientStateResponseDecoded.latestHeight.revisionHeight.toInt());
-    const consensusStateResponseDecoded = decodeTendermintConsensusStateAny(consensusStateResponse.consensusState);
+    const finalResponse = await ibcExtension.ibc.channel.clientState(port, channel).then(async (clientStateResponse) => {
+        const clientStateResponseDecoded = decodeTendermintClientStateAny(clientStateResponse.identifiedClientState.clientState);
+        timeoutHeight = {
+            revisionHeight: clientStateResponseDecoded.latestHeight.revisionHeight.add(1000000),
```
Comment (puneet2019) [inline src/utils/transactions.js:161]: can you take 1000000 from config?

File: src/utils/transactions.js [hunk 2]
```diff
@@ -155,20 +155,23 @@ async function MakeIBCTransferMsg(channel, fromAddress, toAddress, amount, timeo
...
+            revisionNumber: clientStateResponseDecoded.latestHeight.revisionNumber
+        };
 
-    const timeoutTime = Long.fromNumber(consensusStateResponseDecoded.timestamp.getTime()/1000).add(timeoutTimestamp).multiply(1000000000); //get time in nanoesconds
+        const consensusStateResponse = await ibcExtension.ibc.channel.consensusState(port,channel,
+            clientStateResponseDecoded.latestHeight.revisionNumber.toInt() , clientStateResponseDecoded.latestHeight.revisionHeight.toInt());
+        const consensusStateResponseDecoded = decodeTendermintConsensusStateAny(consensusStateResponse.consensusState);
 
-    return TransferMsg(channel, fromAddress, toAddress, amount,timeoutHeight , timeoutTime, denom, port);
+        const timeoutTime = Long.fromNumber(consensusStateResponseDecoded.timestamp.getTime()/1000).add(timeoutTimestamp).multiply(1000000000); //get time in nanoesconds
```
Comment (puneet2019) [inline src/utils/transactions.js:169]: timeoutTimestamp default value from config as well?

**Review by puneet2019 — CHANGES_REQUESTED** [review]

**Review by deepanshutr — DISMISSED** [review]
Reviewed.

**Review by deepanshutr — APPROVED** [review]
Reviewed.

**Review by puneet2019 — APPROVED** [review]

**Pull request merged**